src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -62,10 +63,51 @@ class BaseSidebarScreen extends Component {
         Welcome.show({routes, showCreateMenu: this.showCreateMenu});
     }
 
+    componentDidUpdate(prevProps) {
+        if (!this.didBecomeInactive(prevProps)) {
+            return;
+        }
+
+        // Hide menu manually when other pages are opened using shortcut key
+        this.hideCreateMenu();
+    }
+
+    didBecomeInactive(prevProps) {
```

Comment:
Same for it. A better name is needed. Functions are named on what they do not where they are called.

Reply:
Honestly I'd rather not negate a negative - we're checking if the outcome of this function is false (`!this.didBecomeInactive(prevProps)`) and the function name is about something being "inactive" - it's clearer to check if something is active (instead of _not inactive_)

So how about this name: `checkSidebarScreenNewlyVisible`? I know it's a long name but explains the purpose of the function, I believe